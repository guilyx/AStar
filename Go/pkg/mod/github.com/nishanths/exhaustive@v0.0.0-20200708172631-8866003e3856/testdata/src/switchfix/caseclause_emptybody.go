package switchfix

import (
	bar "switch/y"
)

func _caseclause_emptybody() {
	var phy bar.Phylum
	switch phy { // want "missing cases in switch of type bar.Phylum: Chordata, Echinodermata, Mollusca"
	}
}
