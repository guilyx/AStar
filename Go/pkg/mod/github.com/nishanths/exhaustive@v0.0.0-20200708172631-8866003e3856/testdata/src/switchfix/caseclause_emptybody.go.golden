package switchfix

import (
	"fmt"
	bar "switch/y"
)

func _caseclause_emptybody() {
	var phy bar.Phylum
	switch phy { // want "missing cases in switch of type bar.Phylum: Chordata, Echinodermata, Mollusca"
	case bar.Chordata, bar.Echinodermata, bar.Mollusca:
		panic(fmt.Sprintf("unhandled value: %v", phy))
	}
}
